docs/DeveloperGuide.md

```diff
@@ -557,3 +557,66 @@ testers are expected to do more *exploratory* testing.
     1. _{explain how to simulate a missing/corrupted file, and the expected behavior}_
 
 1. _{ more test cases …​ }_
+
+--------------------------------------------------------------------------------------------------------------------
+
+## **Appendix: Effort**
+
+The creation of GreenMileageEfforts came with a great deal of effort from each and every one of the team members. Since the start of the project we have
+made the effort to consistently conduct teams meeting every week, most of these meetings are held in a physical setting such that we could communicate our ideas effectively.
+Upon completion of the project, we have achieved a product that is a direct reflection of our hard work and determination.
+GreenMileageEfforts is a product that has been adapted and evolved from AB3, where there have been significant enhancements made since AB3.
```

Comment:
should we maybe used the word morph instead of evolve since that was a specific word they used at the start